cpp/src/io/comp/common_internal.hpp

```diff
@@ -40,8 +40,52 @@ constexpr uint8_t fcomment = 0x10;  // Comment present
 constexpr size_t default_host_compression_auto_threshold = 128;
 // Threshold for buffer count where device decompression is favored over host in AUTO mode
 constexpr size_t default_host_decompression_auto_threshold = 128;
+// Estimated ratio between total CPU decompression throughput and decompression throughput of a
+// single GPU block; higher values lead to more host decompression in HYBRID mode
+constexpr double default_host_decompression_ratio = 100;
+// Estimated ratio between total CPU compression throughput and compression throughput of a
+// single GPU block; higher values lead to more host compression in HYBRID mode
+constexpr double default_host_compression_ratio = 100;
```

Comment:
Can't suggest a better name for this but `xx_(de)compression_ratio` initially reads like the ratio between the compressed data size/uncompressed size or vice versa 😄

Reply:
I just made the names similar to `default_host_(de)compression_auto_threshold` above.
Having an accurate name is very hard here, because of how the value is used. I'll think about it some more.
The only accurate name I can think of ATM is `hybrid_(de)compression_fudge_factor` :D

Reply:
I don't know if we can do better either, so that's fine with me.